Makefile

```
@@ -347,22 +347,28 @@ kind-operator-down: $(KIND)
 
 # speed-up skaffold deployments by building all images concurrently
 export SKAFFOLD_BUILD_CONCURRENCY = 0
-gardener%up gardener%dev gardenlet%up gardenlet%dev operator-up operator-dev: export SKAFFOLD_DEFAULT_REPO = localhost:5001
-gardener%up gardener%dev gardenlet%up gardenlet%dev operator-up operator-dev: export SKAFFOLD_PUSH = true
+gardener%up gardener%dev gardenlet%up gardenlet%dev operator-up operator-dev gardener%debug gardenlet%debug operator-debug: export SKAFFOLD_DEFAULT_REPO = localhost:5001
```

Can you sort the targets consistently?
```suggestion
gardener%up gardener%dev gardener%debug gardenlet%up gardenlet%dev gardenlet%debug operator-up operator-dev  operator-debug: export SKAFFOLD_DEFAULT_REPO = localhost:5001
```

---

Done 
I even found some more statements to sort 😄